ui/js/pods-dfv/_src/core/iframe-frame.js

```diff
@@ -28,9 +28,6 @@ export const IframeFrame = wp.media.view.Frame.extend( {
 
 		this.on( 'title:create:default', this.createTitle, this );
 		this.title.mode( 'default' );
-		this.on( 'title:render', function ( view ) {
```

Comment:
@JoryHogeveen Is this only a WP 5.3 thing? Does it work fine for our min WP version (4.5 right now, but it's upped for 2.8)?

Reply:
Pre WP 5.3 it's not visible either (The icon is made invisible in core CSS). I'm guessing it's left-over code within WP.